openshift_performance/ose3_perf/scripts/build_test.py

```diff
@@ -360,8 +363,7 @@ def start():
                 min_all_pushes = global_build_stats[idx]["min_push"]
 
             total_builds += global_build_stats[idx]["num"]
-        else:
-            logger.info("\tNo successful builds")
```

Comment:
NIP: we could change this to logger.debug("\tNo successful builds") ... thus ... it is logged only on the file, not standard output.

Reply:
Good point.   Now logging a single debug line for each build skipped/not run.